### ISSUE TITLE
Package install of v0.4 does not work

### ISSUE THREAD

**Issue by trevorloader** [ryanlewis/seo-metadata#25]
Hey Ryan,

Love your package, but the latest version 0.4 does not install properly into a clean v7.4.3 site.  After installing the zip via the Developer > Packages, when you go to create the data type, Epiphany SEO is not in the list of available data types.

I tried, restarting the website etc, but nothing made it appear.

So I uninstalled and went back to version 0.2.1 which installs and works perfectly.

Thanks
Trevor

**Comment by swollaert** [comment]
I have the same issue with v7.4.1
I installed the package through the package repository.

**Comment by ryanlewis** [comment]
This was changed in recent versions from **Epiphany SEO Metadata** to **SEO Metadata** in the dropdown. The alias renamed unchanged.

Could you check again for the available data types for **SEO Metadata**?

**Comment by swollaert** [comment]
I have reinstalled package version 0.4
![image](https://cloud.githubusercontent.com/assets/13997841/17519147/f9576f9a-5e4a-11e6-985e-4cf9a235619f.png)

In the Property Editor dropdown I can' select 'SEO Metadata'
![image](https://cloud.githubusercontent.com/assets/13997841/17519179/1806e3e4-5e4b-11e6-8838-962df98c776e.png)

**Comment by ryanlewis** [comment]
Aha - it looks like you have installed it as an Umbraco package - did you also install it this way @trevorloader ?

I usually use Nuget to install this package - maybe something has changed, I'll have to take a look. In the meantime, if you are using Visual Studio you can install it via Nuget with 

```
Install-Package Epiphany.SeoMetadata
```

**Comment by swollaert** [comment]
Thanks for checking it out. I'm not using Visual Studio at the moment.

I'm running our umbraco website on azure.
Do you know a guide to setup umbraco development with Visual Studio?

**Comment by ryanlewis** [comment]
[This guide](https://our.umbraco.org/documentation/getting-started/setup/install/install-umbraco-with-nuget) is considered the most up to date.

I would always recommend managing projects this way - it allows you to update Umbraco easily, and many other extensions now prefer to use Umbraco.

**Comment by swollaert** [comment]
Thanks! I'll have a look

**Comment by trevorloader** [comment]
Yes, I installed the package file via the umbraco back end.  And is definitely not installing properly.  If you are not supporting the package install, I'd remove it from our.umbraco.  Cheers, T

**Comment by ryanlewis** [comment]
Closing — this repository is archived. The package targets Umbraco 7, which has been unsupported upstream for years.
